src/sas/sasgui/perspectives/fitting/gpu_options.py

```diff
@@ -167,7 +167,13 @@ def _get_clinfo(self):
             import pyopencl as cl
             platforms = cl.get_platforms()
         except ImportError:
-            warnings.warn("pyopencl import failed. Using only CPU computations")
+            warnings.warn("Unable to import the pyopencl package.  It may not \
+            have been installed.  If you wish to use OpenCL, try running pip \
+            install --user pyopencl")
+        except cl.LogicError as e:
+            warnings.warn("Unable to fetch the OpenCL platforms.  This likely \
+            means that the opencl drivers for your system are not installed.")
+            warnings.warn(e)
```

Comment:
The spaces on the beginning of the line following the backslash will be included in the string.  You can instead do
```
    warn("part 1 "
              "part 2 "
              "part 3")
```
and the python interpreter will join them into a single string.  You can include ```+``` characters between the string parts; python does constant folding in both cases so they are compiled into a single value.

Check that the warnings are showing up in sasview logs and in the GUI.  May want to instead define the logger at the top of the file:
```
import logging
logger = logging.getLogger(__name__)
```
and use:
```
     logger.warn(e)
```